pontoon/insights/static/js/insights_charts.js

```diff
@@ -43,25 +43,13 @@ var Pontoon = (function (my) {
       // Legend configuration doesn't allow for enough flexibility,
       // so we build our own legend
       // eslint-disable-next-line no-unused-vars
-      customLegend: function (chart) {
-        return function (chart) {
-          function renderLabels(chart) {
-            return chart.data.datasets
+      customLegend: (chart) => (chart) => (chart) => chart.data.datasets
```

Comment:
Please review this in an editor. My earlier code suggestions were intended as indicative, not complete. At least this function will now produce a syntax error, and the other string concatenations are untouched. Also, I'd missed on the first pass that the function returned from the outermost function internally calls the innermost one, so that at least should be factored out.